File: ts/wam/1250.ts
export const WamStatusRevoke = 1250 as const

export interface StatusRevokeEvent {
  /** field 1, wire `status_session_id` */
  statusSessionId?: number
  /** field 2, wire `media_type` */
  mediaType?: MEDIATYPE
  /** field 3, wire `status_life_t` */
  statusLifeT?: number
  /** field 4, wire `status_id` */
  statusId?: string
  /** field 5, wire `unified_session_id` */
  unifiedSessionId?: string
  /** field 6, wire `updates_tab_session_id` */
  updatesTabSessionId?: number
}

File: go/wam/1250.go
const WamStatusRevoke = 1250 // channel: regular

type StatusRevokeEvent struct {
	StatusSessionId int64 `wam:"status_session_id"` // field 1
	MediaType MEDIATYPE `wam:"media_type"` // field 2
	StatusLifeT int64 `wam:"status_life_t"` // field 3
	StatusId string `wam:"status_id"` // field 4
	UnifiedSessionId string `wam:"unified_session_id"` // field 5
	UpdatesTabSessionId int64 `wam:"updates_tab_session_id"` // field 6
}